examples/http3-server.rs

```diff
@@ -331,6 +331,19 @@ fn main() {
                                 &headers,
                                 args.get_str("--root"),
                             );
+
+                            // We decide the response based on headers alone.
+                            // Stop reading the stream so that a request
+                            // containing a body does not generate pointless
+                            // Data events.
+                            client
```

Comment:
Might be better to put this directly in `handle_request()` to avoid the weird indentation.

Reply:
Yeah